### DISCUSSION TITLE
external trigger #727

### DISCUSSION THREAD

**Discussion by thomasFrosch** [jomjol/AI-on-the-edge-device#1035] [Q&A]
Hi,

I have seen, that there was a done request to trigger the measurement by external HTML Request.

It looks like issue #727 was actually done, but how can I use it?

What page/url to request, to trigger the measurement?

Thanks!

**Comment by jomjol** [comment]
No - not has been solved. Is in the list for feature request for future releases. Currently no actions on that due to capacity limits.
